README.md

```diff
@@ -111,4 +111,9 @@ https://github.com/dxw/dfsseta-apply-for-landing-ruby/blob/main/.github/workflow
 [deployed to Heroku]:
 https://apply-for-landing-ruby-4492c2b72668.herokuapp.com/
 
+### Environment variables 
 
+The following environment variables must be set on Heroku;
+
+- `HOSTNAME`: currently `apply-for-landing-ruby-4492c2b72668.herokuapp.com`
```

Comment:
I think that where, in the commit message, you say

> but for the dev environment we referr to a HOSTNAME

we actually want to talk about the deployed production environment at Heroku. We aren't using the term "dev" to refer to that.